File: bs/src/main/java/com/union/bangbang/bs/ui/user/login/LoginActivity.java
package com.union.bangbang.bs.ui.user.login;

import android.os.Bundle;
import android.os.PersistableBundle;
import android.util.Log;

import androidx.annotation.Nullable;

import com.union.bangbang.base.base.BaseActivity;
import com.union.bangbang.base.base.BaseModel;
import com.union.bangbang.bs.R;

import javax.inject.Inject;

/**
 * 不乱于心，不困于情。不畏将来，不念过往。如此，安好!
 * <p>
 * 深谋若谷，深交若水。深明大义，深悉小节。已然，静舒!
 * <p>
 * 善宽以怀，善感以恩。善博以浪，善精以业。这般，最佳!
 * <p>
 * 勿感于时，勿伤于怀。勿耽美色，勿沉虚妄。从今，进取!
 * <p>
 * 无愧于天，无愧于地。无怍于人，无惧于鬼。这样，人生!
 */
public class LoginActivity extends BaseActivity {
    @Inject BaseModel baseModel;
    private String TAG ="LoginActivity";
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.i(TAG, "onCreate: "+baseModel.getS());
    }

    @Override
    protected int getLayoutId() {
        return R.layout.activity_login;
    }
}

File: todokotlin/src/main/java/com/union/bangbang/todokotlin/ui/user/login/LoginActivity.kt
package com.union.bangbang.todokotlin.ui.user.login

import android.os.Bundle
import android.view.Gravity
import com.alibaba.android.arouter.facade.annotation.Route
import com.union.bangbang.todokotlin.R
import com.union.bangbang.todokotlin.base.activity.BaseActivity
import com.union.bangbang.todokotlin.dagger.module.ActivityModule.Companion.user_login
import com.union.bangbang.todokotlin.databinding.ActivityLoginBinding
import javax.inject.Inject

/**
 * Rabies
 * @author USER
 * Date:   2019-01-21
 * Time:   13:49
 */
@Route(path = user_login)
class LoginActivity : BaseActivity<ActivityLoginBinding>() {


    @Inject
    lateinit var viewModel: LoginModel

    override fun getLayoutId(): Int = R.layout.activity_login
    override fun initViewModel(): LoginModel {
        return viewModel
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        slidingDirection = Gravity.BOTTOM
        super.onCreate(savedInstanceState)
    }
}